package.json

```diff
@@ -34,7 +34,7 @@
     "test": "jest test && cd examples/2-scaffolding && yarn && yarn start",
     "watch": "jest test --watch",
     "build": "microbundle --no-compress --external mobx,mobx-react,mobx-state-tree,graphql-tag,graphql,react,react-dom,graphql-request",
-    "format": "prettier --write ./**/*"
+    "format": "prettier --write '**/*'"
```

Comment:
This line!

So removing the `./` bit magically fixed the problems I was having running prettier on some directories.

The single-quotes around `**/*` isn't Windows compatible because cmd.exe doesn't strip quotes from around command-line arguments. But I know the quotes are desired in linux so that the glob pattern is passed as-is to prettier command, and not expanded by the shell into megabytes of filenames from node_modules. So for now I'll just patch my local working copy here.

Reply:
yeah, for some reason without the quotes prettier ignores `examples` for example. I have no idea why...